pallets/asset/src/tests.rs

```diff
@@ -495,3 +495,62 @@ fn asset_vc_status_change_should_succeed() {
 		));
 	});
 }
+
+#[test]
+fn asset_create_unauthorized_operation() {
```

Comment:
Change method name to `asset_functionalities_with_invalid_authorization_should_fail`